### DISCUSSION TITLE
Weird Error: Importing binding name 'default' cannot be resolved by star export entries.

### DISCUSSION THREAD

**Discussion by HuakunShen** [nuxt/nuxt#28388] [Questions]
I am getting this error

```
Importing binding name 'default' cannot be resolved by star export entries.

createError@http://localhost:3000/_nuxt/@fs/Users/user/Dev/projects/app/node_modules/.pnpm/h3@1.12.0/node_modules/h3/dist/index.mjs:78:26
createError@http://localhost:3000/_nuxt/@fs/Users/user/Dev/projects/app/node_modules/.pnpm/nuxt@3.12.3_@opentelemetry+api@1.9.0_@parcel+watcher@2.4.1_@types+node@20.14.12_encoding@0.1._rjcuoiujgjgjgxwpycebyyys54/node_modules/nuxt/dist/app/composables/error.js:33:34
showError@http://localhost:3000/_nuxt/@fs/Users/user/Dev/projects/app/node_modules/.pnpm/nuxt@3.12.3_@opentelemetry+api@1.9.0_@parcel+watcher@2.4.1_@types+node@20.14.12_encoding@0.1._rjcuoiujgjgjgxwpycebyyys54/node_modules/nuxt/dist/app/composables/error.js:9:32
runWithContext@http://localhost:3000/_nuxt/@fs/Users/user/Dev/projects/app/node_modules/.pnpm/@vue+runtime-core@3.4.33/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js:2678:20
run@http://localhost:3000/_nuxt/@fs/Users/user/Dev/projects/app/node_modules/.pnpm/@vue+reactivity@3.4.33/node_modules/@vue/reactivity/dist/reactivity.esm-bundler.js:32:18
executeAsync@http://localhost:3000/_nuxt/@fs/Users/user/Dev/projects/app/node_modules/.pnpm/unctx@2.3.1/node_modules/unctx/dist/index.mjs:111:28
@http://localhost:3000/_nuxt/@fs/Users/user/Dev/projects/app/node_modules/.pnpm/nuxt@3.12.3_@opentelemetry+api@1.9.0_@parcel+watcher@2.4.1_@types+node@20.14.12_encoding@0.1._rjcuoiujgjgjgxwpycebyyys54/node_modules/nuxt/dist/pages/runtime/plugins/router.js:128:43
```

After debugging I found that the error comes from [Nuxt MDC](https://nuxt.com/modules/mdc) ([0.8.3](https://www.npmjs.com/package/@nuxtjs/mdc/v/0.8.3)). 
If I add `<MDC/>` component I get the error. But I used it for a long time and had no problem before.

Then I did a git bisect and found that the error appears after I add [i18n nuxt module](https://i18n.nuxtjs.org/docs/getting-started) [8.3.3](https://www.npmjs.com/package/@nuxtjs/i18n/v/8.3.3). 

What's weird is, when I want to make a minimum reproduction with the 2 modules in a new nuxt project everything works fine.

I have multiple other nuxt modules installed, not sure if they affect each other, but I couldn't make a minimum reproduction for now.

Anyone seen this error or know what's going on? I have seen this multiple times with Nuxt.

BTW, I get this error only in dev mode. The app builds and could run in production.

**Comment by HuakunShen** [comment]
Due to the auto import feature, I also don't know how to manually debug this problem.

Removing i18n module from nuxt config doesn't get rid of the error, I have to completely delete the entire node_modules and reinstall everything, then the error is gone. 
No clue where to start debugging.

**Comment by jswhisperer** [comment]
This is hella confusing error, probably not a nuxt issue but more a VITE module resolution thing but so hard to find the offending packages...

Got the same error importing "marked"
